gk/main.c

```diff
@@ -634,6 +711,34 @@ gk_proc(void *arg)
 			process_gk_cmd(gk_cmds[i], instance);
 			mb_free_entry(&instance->mb, gk_cmds[i]);
         	}
+
+		if (ipv6_if_configured(&gk_conf->net->front) && acl.num > 0) {
+			ret = rte_acl_classify(
```

Comment:
I see the `__attribute__((constructor))` now on `rte_acl_init()`, so I agree that the ACL library will take care of it -- we can remove the `XXX` about changing the classification.